android/README.md

```diff
@@ -0,0 +1,456 @@
+# Developer Documentation - For Android SDK
+
+## Introduction
+
+The Asgardeo Auth Android SDK enables Android applications (written in Kotlin) to utilize OpenID Connect (OIDC) authentication with the WSO2 Identity Server/Asgardeo serving as the Consumer Identity and Access Management (CIAM) Provider through API-based authentication. This SDK assists you in creating custom login flows directly within the applications themselves, without relying on browser redirects, thereby prioritizing user experience.
```

Comment:
Kotlin libs can be used from Java Android apps as well, right? If yes, do we need to say that this sdk is only for apps written in Kotlin?

```suggestion
The Asgardeo Auth Android SDK enables Android applications to utilize OpenID Connect (OIDC) authentication with the WSO2 Identity Server/Asgardeo serving as the Consumer Identity and Access Management (CIAM) Provider through API-based authentication. This SDK assists you in creating custom login flows directly within the applications themselves, without relying on browser redirects, thereby prioritizing user experience.
```

Reply:
Yes you are correct, but using Kotlin libs in Java projects is bit cumbersome effort and not recommend by many, hence its better to mention the technology we have used for the developers information.